### PR TITLE
[ET-1043] Fixed showing notices for localize script on attendee list page

### PR DESCRIPTION
**Pull request by rafsuntaskin**
🎫 [ET-1043]

Showing this error on the Attendee List page for Post Types:

```
[11-Mar-2021 18:56:10 UTC] PHP Notice:  WP_Scripts::localize was called <strong>incorrectly</strong>. The <code>$l10n</code> parameter must be an array. To pass arbitrary data to scripts, use the <code>wp_add_inline_script()</code> function instead. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 5.7.0.) in /app/public/wp-includes/functions.php on line 5313
[11-Mar-2021 18:56:10 UTC] PHP Stack trace:
[11-Mar-2021 18:56:10 UTC] PHP   1. {main}() /app/public/wp-admin/edit.php:0
[11-Mar-2021 18:56:10 UTC] PHP   2. require_once() /app/public/wp-admin/edit.php:10
[11-Mar-2021 18:56:10 UTC] PHP   3. require_once() /app/public/wp-admin/admin.php:239
[11-Mar-2021 18:56:10 UTC] PHP   4. do_action() /app/public/wp-admin/admin-header.php:102
[11-Mar-2021 18:56:10 UTC] PHP   5. WP_Hook->do_action() /app/public/wp-includes/plugin.php:484
[11-Mar-2021 18:56:10 UTC] PHP   6. WP_Hook->apply_filters() /app/public/wp-includes/class-wp-hook.php:316
[11-Mar-2021 18:56:10 UTC] PHP   7. Tribe__Tickets__Attendees->load_pointers() /app/public/wp-includes/class-wp-hook.php:292
[11-Mar-2021 18:56:10 UTC] PHP   8. wp_localize_script() /app/public/wp-content/plugins/event-tickets/src/Tribe/Attendees.php:368
[11-Mar-2021 18:56:10 UTC] PHP   9. WP_Scripts->localize() /app/public/wp-includes/functions.wp-scripts.php:221
[11-Mar-2021 18:56:10 UTC] PHP  10. _doing_it_wrong() /app/public/wp-includes/class.wp-scripts.php:496
[11-Mar-2021 18:56:10 UTC] PHP  11. trigger_error() /app/public/wp-includes/functions.php:5313
[11-Mar-2021 18:56:10 UTC] PHP Notice:  WP_Scripts::localize was called <strong>incorrectly</strong>. The <code>$l10n</code> parameter must be an array. To pass arbitrary data to scripts, use the <code>wp_add_inline_script()</code> function instead. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 5.7.0.) in /app/public/wp-includes/functions.php on line 5313
[11-Mar-2021 18:56:10 UTC] PHP Stack trace:
[11-Mar-2021 18:56:10 UTC] PHP   1. {main}() /app/public/wp-admin/edit.php:0
[11-Mar-2021 18:56:10 UTC] PHP   2. require_once() /app/public/wp-admin/edit.php:10
[11-Mar-2021 18:56:10 UTC] PHP   3. require_once() /app/public/wp-admin/admin.php:239
[11-Mar-2021 18:56:10 UTC] PHP   4. do_action() /app/public/wp-admin/admin-header.php:102
[11-Mar-2021 18:56:10 UTC] PHP   5. WP_Hook->do_action() /app/public/wp-includes/plugin.php:484
[11-Mar-2021 18:56:10 UTC] PHP   6. WP_Hook->apply_filters() /app/public/wp-includes/class-wp-hook.php:316
[11-Mar-2021 18:56:10 UTC] PHP   7. {closure:/app/public/wp-content/plugins/event-tickets/common/vendor/lucatume/di52/src/tad/DI52/closuresSupport.php:28-32}() /app/public/wp-includes/class-wp-hook.php:292
[11-Mar-2021 18:56:10 UTC] PHP   8. Tribe__Tickets__Attendees->load_pointers() /app/public/wp-content/plugins/event-tickets/common/vendor/lucatume/di52/src/tad/DI52/closuresSupport.php:31
[11-Mar-2021 18:56:10 UTC] PHP   9. wp_localize_script() /app/public/wp-content/plugins/event-tickets/src/Tribe/Attendees.php:368
[11-Mar-2021 18:56:10 UTC] PHP  10. WP_Scripts->localize() /app/public/wp-includes/functions.wp-scripts.php:221
[11-Mar-2021 18:56:10 UTC] PHP  11. _doing_it_wrong() /app/public/wp-includes/class.wp-scripts.php:496
[11-Mar-2021 18:56:10 UTC] PHP  12. trigger_error() /app/public/wp-includes/functions.php:5313
```

[ET-1043]: https://theeventscalendar.atlassian.net/browse/ET-1043

### CODE REVIEW

**Comment by bordoni** [comment]
Needs a changelog.

**Review by sc0ttkclark — APPROVED** [review]

**Comment by sc0ttkclark** [comment]
Needs changelog

File: readme.txt [hunk 1]
```diff
@@ -179,6 +179,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 = [5.1.2] TBD =
 
 * Fix - Add "Currently full" message on TEC views when the event has RSVPs without availability. [ET-1004]
+* Fix - Fixed showing notices for localized script in the attendee list page. [ET-1043]
```
Comment (sc0ttkclark) [inline readme.txt:182]: Should this have been Attendees report page?

**Review by sc0ttkclark — DISMISSED** [review]
Looks good, but maybe a tweak if you think it's necessary

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**